simplq/src/store/asyncActions/updateQueueStatus.js

```diff
@@ -0,0 +1,32 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { useMakeAuthedRequest } from 'api/auth';
+import * as RequestFactory from 'api/requestFactory';
+import { useHistory } from 'react-router';
+
+const typePrefix = 'updateQueueStatus/action';
+
+/**
+ * A hook to access the updateQueueStatus async action creator.
+ *
+ * @returns — updateQueueStatus async action creator
+ */
+const useUpdateQueueStatus = () => {
```

Comment:
Hey, a small request. Can you do a Find/Replace for "updateQueueStatus" in this PR with "setQueueStatus".

I think set/get is better naming. In the RequestFactory, you've used "setQueueStatus"